admin-dev/themes/new-theme/js/pages/order/create/create-order-page.js

```diff
@@ -66,6 +66,34 @@ export default class CreateOrderPage {
     this._initListeners();
   }
 
+  /**
+   * Checks if correct addresses are selected.
+   * There is a case when options list cannot contain cart addresses 'selected' values
+   *  because those are outdated in db (e.g. deleted after cart creation or country is disabled)
+   *
+   * @param {Array} addresses
+   *
+   * @returns {boolean}
+   */
+  static validateSelectedAddresses(addresses) {
+    let deliveryValid = false;
+    let invoiceValid = false;
+
+    for (const key in addresses) {
+      const address = addresses[key];
+
+      if (address.delivery) {
+        deliveryValid = true;
+      }
+
+      if (address.invoice) {
+        invoiceValid = true;
+      }
+    }
+
+    return deliveryValid && invoiceValid;
```

Comment:
the return must be in the for loop when `deliveryValid` and `invoiceValid`. Otherwise if you have 100 addresses and the first one is valid, you loop the whole array for nothing.